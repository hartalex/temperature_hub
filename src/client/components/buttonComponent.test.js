import React from 'react';
import ButtonComponent from './buttonComponent.js';
import renderer from 'react-test-renderer';

describe('Button Component', async () => {
  it('Renders', async () => {
    const component = renderer.create(<ButtonComponent />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
