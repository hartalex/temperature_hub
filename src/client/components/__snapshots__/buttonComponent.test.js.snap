// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Renders 1`] = `
<div
  style={
    Object {
      "background": "#000000",
      "border": "1px solid darkgray",
      "color": "#FFFFFF",
      "float": "left",
      "fontSize": "10px",
      "height": "125px",
      "textAlign": "center",
      "width": "127px",
    }
  }
>
  <div
    style={
      Object {
        "padding": "2px 0",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "auto",
          "width": "100px",
        }
      }
    >
      <div
        style={
          Object {
            "float": "left",
            "fontSize": "32px",
          }
        }
      >
        0
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": "7px",
        "textAlign": "right",
      }
    }
  >
    21218 hour
  </div>
</div>
`;
